cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -1085,7 +1142,7 @@ and [2.5.1](<#head2.5.1>).
 -   The value of ticket-granting cookies SHALL contain adequate secure random data
     so that a ticket-granting cookie is not guessable in a reasonable period of time.
 
--   The value of ticket-granting cookies SHOULD begin with the characters "TGC-".
+-   The value of ticket-granting cookies SHOULD begin with the characters `TGC-`.
```

Comment:
We're using CASTGC for the name of the cookie, not the prefix of the value, correct?  The prefix of the value is TGT.